packages/image-react/src/Image.tsx

```diff
@@ -1,59 +1,60 @@
-import React, { FC } from "react";
-import classNames from "classnames";
-import { useProgressiveImg } from "@fremtind/jkl-react-hooks";
+import React, { VFC } from "react";
+import cn from "classnames";
+import { useElementDimensions, useImageLoadingStatus } from "@fremtind/jkl-react-hooks";
 
-export interface Props {
-    defaultSize: string;
-    isSmallDevice: boolean;
+export interface ProgressiveImage {
+    src: string;
+    /**
+     * En liste med bilder for forskjellig oppløsning/størrelse
+     * https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/srcset
+     */
+    srcSet: string;
+    /**
+     * Bilde med veldig lav oppløsning som vises uskarpt under innlasting.
+     * Anbefalt maks: 50px langside, 3-5 kB filstørrelse
+     */
+    placeholder?: string;
+}
+
+export const isProgressiveImage = (image: string | ProgressiveImage): image is ProgressiveImage =>
+    typeof (image as ProgressiveImage).srcSet !== "undefined";
+
+export interface ImageProps {
+    src: string | ProgressiveImage;
+    /**
+     * Send inn en tom streng hvis bildet kun er dekorativt.
```

Comment:
Kanskje og skrive et lite hint om at bilder [sannsynligvis ikke er dekorative](https://www.smashingmagazine.com/2021/06/img-alt-attribute-alternate-description-decorative/)

```suggestion
     * Send inn en tom streng hvis bildet kun er dekorativt. Sannsynligvis bør du beskrive innholdet i bildet så en person med nedsatt syn får en fullgod opplevelse av siden.
```